README.md

```diff
@@ -12,9 +12,27 @@
 </p>
 
 <p align="center">
-  <img src="assets/screenshot.webp"/>
+  <img src="https://raw.githubusercontent.com/catppuccin/alacritty/main/assets/preview.webp"/>
```

Comment:
I believe that hard-linking the preview images is a mistake. The preview on the template repo is only hard-linked because the image itself is from a completely different repo altogether.

Port-readmes usually just softlink the previews. c:
```markdown

<p align="center">
  <img src="assets/preview.webp"/>
</p>

## Previews

<details>
  <summary>🌻 Latte</summary>
  <img src="assets/latte.webp"/>
</details>
<details>
  <summary>🪴 Frappé</summary>
  <img src="assets/frappe.webp"/>
</details>
<details>
  <summary>🌺 Macchiato</summary>
  <img src="assets/macchiato.webp"/>
</details>
<details>
  <summary>🌿 Mocha</summary>
  <img src="assets/mocha.webp"/>
</details>
```

Reply:
I've heard both over time, we should declare that somewhere, but changes are on their way non-the less :)